packages/node_modules/node-red/settings.js

```diff
@@ -416,6 +416,11 @@ module.exports = {
              * If not set, the following default order is used:
              */
             //categories: ['subflows', 'common', 'function', 'network', 'sequence', 'parser', 'storage'],
+
+            /**
+             * To enable checking for updates every 30min when the editor is open and active.
+             */
+            checkForUpdates: false,
```

Comment:
Seeing as you are adding a setting - why not make this the actual interval instead of just true/false... If set to a valid number then use it - if not or false then no update check.